tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/FilterPushDownRule.java

```diff
@@ -235,9 +232,50 @@ public LogicalNode visitJoin(FilterPushDownContext context, LogicalPlan plan, Lo
     }
 
     context.pushingDownFilters.removeAll(matched);
+
+    if (context.pushingDownFilters.size() > 0) {
```

Comment:
It would be great if you add some comments to explain its logical meaning about the condition and what this code do.